src/call.rs

```diff
@@ -245,14 +244,40 @@ async fn query_account_boc(ton: TonClient, addr: &str) -> Result<String, String>
     Ok(boc.unwrap().to_owned())
 }
 
+pub async fn emulate_localy(
+    ton: TonClient,
+    addr: &str,
+    msg: String,
+) -> Result<(), String> {
+    let state_boc = query_account_boc(ton.clone(), addr).await?;
+
+    let res = run_executor(
+        ton.clone(),
+        ParamsOfRunExecutor {
+            message: msg.clone(),
+            account: AccountForExecutor::Account {
+                boc: state_boc,
+                unlimited_balance: None,
+            },
+            ..Default::default()
+        },
+    )
+    .await;
+
+    if res.is_err() {
+        return Err(format!("Local execution has failed:\n{:#}", res.err().unwrap()));
```

Comment:
I tried it out and in my opinion we should remove this text: `Local execution has failed:\n{:#}`, just return original error.

Reply:
Fixed